### DISCUSSION TITLE
How to load .spacy format object into Python? (spacy-nightly)

### DISCUSSION THREAD

**Discussion by yw2903** [explosion/spaCy#7043] [Help: Coding & Implementations] [answered]
How can I load a .spacy format object in python to do some custom evaluation. Though **evaluate** gives the precision and recall, but I would like to see what are the false positives look like.

**Comment by AndriyMulyar** [comment, marked as answer]
The .spacy extension likely refers to a DocBin object that was sent to disk. Try something like [this](https://spacy.io/api/docbin#from_disk).

**Reply by yw2903** [reply]
Thanks, does it matter which model I load as nlp when de-serialize docs?
```
nlp = spacy.load("en_core_web_lg")
docs = list(doc_bin.get_docs(nlp.vocab))
```
